### PR TITLE
Limit unnecessary logging

### PR DESCRIPTION
**Pull request by philipqueen**
This PR intends to reduce the clutter in our logging, especially what is shown in the log view widget:

1. It turns the log view widget logging level from debug to info, so not everything is displayed in the gui
2. Sets a lot of existing logger calls to debug from info
3. Fixes the logging output from the anipose camera calibration

I've also added some TODOs around the calibration toml finding, due to the fact that this is the greatest offender when it comes to logging spam.

I've also deleted some old code that was commented out, and typed some functions that confused me while tracking down issues.

### CODE REVIEW

File: freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py [hunk 1]
```diff
@@ -93,21 +93,22 @@ def calibrate_camera_capture_volume(self, pin_camera_0_to_origin: bool = False):
...
 
         self._anipose_camera_group_object.dump(recording_folder_toml_path)
-        logger.info(f"anipose camera calibration data saved to Recording folder - {str(recording_folder_toml_path)}")
+        logger.debug(f"anipose camera calibration data saved to Recording folder - {str(recording_folder_toml_path)}")
 
         last_successful_calibration_toml_path = get_last_successful_calibration_toml_path()
         self._anipose_camera_group_object.dump(last_successful_calibration_toml_path)
 
-        logger.info(
+        logger.debug(
```
Comment (jonmatthis) [inline freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py:108]:
```suggestion
        logger.info(
```

I think as a general rule, if we should show saving something to disk, we should show the path to the User

Comment (jonmatthis) [reply freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py:108]: (just say that you might be doing his in the `.._progress_callback` below - Your call!

File: freemocap/core_processes/export_data/blender_stuff/export_to_blender/methods/legacy/run_megascript_take2.py [hunk 2]
```diff
@@ -52,17 +52,17 @@ def run_blender_megascript_take2(
         str(blender_file_path),
     ]
 
-    logger.info(f"Starting `blender` sub-process with this command: \n {command_list}")
+    logger.debug(f"Starting `blender` sub-process with this command: \n {command_list}")
```
Comment (jonmatthis) [inline freemocap/core_processes/export_data/blender_stuff/export_to_blender/methods/legacy/run_megascript_take2.py:55]: see above (let me know if "above" makes sense in the medium you wind up reading this)

File: freemocap/gui/qt/widgets/log_view_widget.py [hunk 3]
```diff
@@ -133,6 +146,40 @@ def add_log(self, record: logging.LogRecord):
         )
         self.appendHtml(colored_log_entry)
 
+    @QtCore.Slot()
+    def log_progress(self, bar_length: int = 100, cycles_per_update: int = 2500) -> None:
+        if not hasattr(self, "_progress_counter"):
+            self._progress_counter = 0
+
+        progress_marker = "▁▂▃▄▅▆▇▆▅▄▃▂▁"
```
Comment (jonmatthis) [inline freemocap/gui/qt/widgets/log_view_widget.py:154]: aw cute

File: freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py (hunk 1 above)
Comment (philipqueen) [reply freemocap/core_processes/capture_volume_calibration/anipose_camera_calibration/anipose_camera_calibrator.py:108]: I think the progress callback is good enough here - remember the debug messages will still be saved in the logs and printed to the terminal. Also generally, the calibration toml stuff gets logged in a few different places

File: freemocap/core_processes/export_data/blender_stuff/export_to_blender/methods/legacy/run_cgtinker_method.py [hunk 4]
```diff
@@ -42,7 +42,7 @@ def run_cgtinker_method(
         "1,",  # load quick
     ]
 
-    logger.info(f"Starting `blender` sub-process with this command: \n {command_list}")
+    logger.debug(f"Starting `blender` sub-process with this command: \n {command_list}")
```
[the remark this answers is not in the document]
Comment (philipqueen) [reply freemocap/core_processes/export_data/blender_stuff/export_to_blender/methods/legacy/run_cgtinker_method.py]: Made this change along with the other equivalent ones

File: freemocap/gui/qt/widgets/log_view_widget.py (hunk 3 above)
Comment (philipqueen) [reply freemocap/gui/qt/widgets/log_view_widget.py:154]: I had a lot of fun trying different ASCII art for this

**Pull request merged**